rudder-web/src/main/webapp/templates-hidden/common-layout.html

```diff
@@ -56,55 +56,81 @@
-        <!-- Right Menu -->
-        <ul class="nav navbar-nav navbar-right">
-          <!-- Change request  -->
-          <li class="lift:comet?type=WorkflowInformation" name="workflowInfo" ></li>
-          <!-- Promise Generation -->
-          <li class="lift:comet?type=AsyncDeployment" name="deploymentStatus"  ></li>
-          <!-- Manual -->
-          <li><a href="/rudder-doc" target="_blank">User manual</a></li>
-          <!-- User -->
-          <li class="dropdown">
-            <a href="#" class="dropdown-toggle"  data-toggle="dropdown">
-              <span data-lift="UserInformation.userCredentials">
-                <span style="margin-right:5px;" class="glyphicon glyphicon-user"></span>
-                <span id="openerAccount">[User name]</span>
-                <span class="caret"></span>
-              </span>
-            </a>
-            <ul class="dropdown-menu" role="menu">
-              <li><a href="#" id="logout">Log out</a></li>
+  <div class="wrapper">
+      <header class="main-header">
+        <a href="/secure/" class="logo fixed">
+          <span class="logo-mini"><b>R</b></span>
+          <span class="logo-lg"><b>Rudder</b></span>
+        </a>
+        <nav class="navbar navbar-static-top" role="navigation">
+          <a href="#" class="sidebar-toggle" data-toggle="offcanvas" role="button">
+            <span class="sr-only">Toggle navigation</span>
+          </a>
+          <div class="navbar-custom-menu">
+            <ul class="nav navbar-nav">
+            <!-- Change request  -->
+            <li class="lift:comet?type=WorkflowInformation" name="workflowInfo" ></li>
+            <!-- Promise Generation -->
+            <li class="lift:comet?type=AsyncDeployment" name="deploymentStatus"  ></li>
+              <li class="tasks-menu">
+                <a target="_blank" href="/rudder-doc">
+                  <i class="fa fa-book"></i>
+                  <span class="hidden-xs"> User manual </span>
+                </a>
+              </li>
+              <!-- User -->
+              <li class="user user-menu">
+                <a href="#">
+                  <span data-lift="UserInformation.userCredentials">
+                    <span class="hidden-xs glyphicon glyphicon-user"></span>
+                    <span id="openerAccount">[User name]</span>
+                  </span>
+                </a>
+              </li>
+              <!-- Log out -->
+              <li>
+                <a id="logout" href="#" data-toggle="control-sidebar"><i class="fa fa-power-off"></i></a>
+              </li>
             </ul>
-          </li>
-        </ul>
-      </div>
+          </div>
+        </nav>
+      </header>
+      <aside class="main-sidebar fixed">
+        <section class="sidebar">
+          <!-- Quick Search-->
+         <div data-lift="QuickSearchNode.render" ></div>
+          <!-- Navigation Menu -->
+          <span class="lift:BootstrapMenu?linkToSelf=true;"></span>
```

Comment:
Should be:

<ul class="sidebar-menu">
   <span class="lift:BootstrapMenu?linkToSelf=true;"></span>
   <li class="treeview footer">
     <span>Rudder v.3.3.0</span>
     <a href="https://www.normation.com/fr/" target="_blank">© Normation 2016</a>
   </li>
</ul>